web/src/main/java/org/akaza/openclinica/controller/DicomController.java

```diff
@@ -0,0 +1,58 @@
+package org.akaza.openclinica.controller;
+
+import org.akaza.openclinica.bean.login.UserAccountBean;
+import org.akaza.openclinica.service.DicomServiceClient;
+import org.akaza.openclinica.service.UtilService;
+import org.akaza.openclinica.service.randomize.RandomizationService;
+import org.akaza.openclinica.web.util.ErrorConstants;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.multipart.MultipartFile;
+
+import javax.servlet.http.HttpServletRequest;
+
+/**
+ * Controller to handle DICOM upload.
+ * @author svadla@openclinica.com
+ */
+@Controller
+@RequestMapping(value = "/auth/api/dicom")
+public class DicomController {
+
+    @Autowired
+    private UtilService utilService;
+    @Autowired
+    private DicomServiceClient dicomServiceClient;
+
+    protected final Logger logger = LoggerFactory.getLogger(getClass().getName());
+
+    @RequestMapping(method = RequestMethod.POST, consumes = {"multipart/form-data"})
+    public ResponseEntity<Object> importDataPipeDelimitedFile(HttpServletRequest request,
```

Comment:
Is it possible to return String here instead of object? Both conditions seem to return a string.
If you use RestController, you could use 
public String importData...